src/soap/index.ts

```diff
@@ -60,11 +60,18 @@ export class SoapClient extends BaseClient {
       ...params.additionalRequestOptions
     };
     if (params.overwriteWsdl != null) {
-      return await createClientAsync(params.overwriteWsdl, {});
+      return await createClientAsync(params.overwriteWsdl, undefined, params.overwriteEndpoint);
```

Comment:
what does the `undefined` mean ? maybe to put as a default variable

Reply:
ready!